docs/remote-feature-flags.md

```diff
@@ -0,0 +1,171 @@
+# Remote Feature Flags
+
+## Overview
+
+This document outlines the process for adding and managing remote feature flags in MetaMask (Extension and Mobile). Remote feature flags allow us to control feature availability and behavior in production without requiring a new release. They are created on the Launch Darkly platform and distributed by our internal API [ClientConfigAPI](https://github.com/consensys-vertical-apps/mmwp-client-config-api). Please read the [Remote feature flags ADR](https://github.com/MetaMask/decisions/pull/43) for more details.
+
+## Implementation Guide
+
+### 1. Creating a New Feature Flag
+
+1. Name your flag with team prefix (e.g., `confirmations-blockaid`, `ramps-card`)
+2. Request LaunchDarkly access from TechOps. Here's a template email:
+
+```
+Subject: Requesting access to LaunchDarkly for [team]
+
+Dear TechOps,
+Can I please get access to LaunchDarkly with the "Metamask Extension and Mobile" role? (role key: `metamask-config`)
+
+Thanks!
+```
+
+3. Select appropriate project in LaunchDarkly:
+   - Extension: "MetaMask Client Config API - Extension"
+   - Mobile: "MetaMask Client Config API - Mobile"
+4. Create the flag following the [LaunchDarkly flag creation guide](https://docs.launchdarkly.com/home/flags/new)
+
+### 2. Using Remote Feature Flags in MetaMask
```

Comment:
I'd say `2. Using Remote Feature Flags in MetaMask clients`.

I'd also swap this part with `3. Remote Feature Flags with thresholds` as we first need to create the flag in LaunchDarkly, then implement it in the client.

Reply:
The order now will be 
1. Creating feature flag in LaunchDarkly.
2. Using Remote Feature Flags in MetaMask clients
3. Testing Remote Feature Flags
See 03cd060fbdb2b2274b717491eaf23862aa604239